integration-tests/src/test/resources/features/error-handling.feature

```diff
@@ -124,4 +132,4 @@ Feature: Error handling tests
         }
     }
     """
-    Then Webhook site with id "${env.webhook.site.uuid}" contains request with text ""id":"${cloud-event.error-handling-update-test.id}","source":"mySource","type":"myType","subject":"mySubject"" within 1 minute
\ No newline at end of file
+    Then Kafka topic "topic1" contains message with text of ""id":"${cloud-event.error-handling-update-test.id}","source":"mySource","type":"myType","subject":"mySubject""
```

Comment:
A general note - not sure if we can guarantee that the Kafka message is delivered sooner than this step is invoked. Would it have sense to add a timeout here?